jupyterhub/jupyterhub/base/jupyterhub-singleuser-profiles-configmap.yaml

```diff
@@ -17,6 +17,10 @@ data:
             memory: 2Gi
             cpu: 1
 
+      - name: Elyra Notebook
```

Comment:
We don't need this right now..the image will be listed based on the image streams